### PR TITLE
feat(workflow): Dynamic counts tooltip links to events if user doesn't have discover

### PR DESCRIPTION
**Pull request by taylangocmen**
This modifies the dynamic counts tooltip link so that it links to issue page events tab with the related query. Also added a list of properties that needs to be removed from the query so that the search works both in discover and issue/events.

Resolves: [WOR-259](https://getsentry.atlassian.net/browse/WOR-259)

### CODE REVIEW

File: src/sentry/static/sentry/app/components/stream/group.jsx [hunk 1]
```diff
@@ -130,45 +145,59 @@ const StreamGroup = createReactClass({
...
+      if (!!start && !!end) {
+        discoverQuery.start = start;
+        discoverQuery.end = end;
+      } else {
+        discoverQuery.range = period || DEFAULT_STATS_PERIOD;
+      }
+
+      const discoverView = EventView.fromSavedQuery(discoverQuery);
+      return discoverView.getResultsViewUrlTarget(organization.slug);
     } else {
```
Comment (scttcper) [inline src/sentry/static/sentry/app/components/stream/group.jsx:192]: could probably drop this else now that they both include returns

File: src/sentry/static/sentry/app/components/stream/group.jsx [hunk 2]
```diff
@@ -130,45 +145,59 @@ const StreamGroup = createReactClass({
...
-        if (!['is', '__text'].includes(queryTag)) {
+        if (!DiscoveryExclusionFields.includes(queryTag)) {
           const queryVal = queryObj[queryTag].includes(' ')
             ? `"${queryObj[queryTag]}"`
             : queryObj[queryTag];
-          discoveryQueryTerms.push(`${queryTag}:${queryVal}`);
+          queryTerms.push(`${queryTag}:${queryVal}`);
         }
+
+      if (!hasDiscoverQuery && queryObj.__text) queryTerms.push(queryObj.__text);
```
Comment (scttcper) [inline src/sentry/static/sentry/app/components/stream/group.jsx:163]: we tend to still use brackets for single line statements

**Review by scttcper — APPROVED** [review]

File: src/sentry/static/sentry/app/components/stream/group.jsx (hunk 2 above)
Comment (billyvg) [reply src/sentry/static/sentry/app/components/stream/group.jsx:163]: Can someone look into adding this as a lint rule?

Comment (taylangocmen) [reply src/sentry/static/sentry/app/components/stream/group.jsx:163]: Thanks

File: src/sentry/static/sentry/app/components/stream/group.jsx (hunk 1 above)
Comment (taylangocmen) [reply src/sentry/static/sentry/app/components/stream/group.jsx:192]: Yeah I'll do that

**Pull request merged**